src/App.vue

```diff
@@ -1,19 +1,30 @@
 <template>
   <div id="app">
-    <b-navbar toggleable="lg" type="dark" variant="info">
+    <b-container>
+      <b-row>
+        <b-col>
+          <div class="mt-2">
+            <router-link to="/">
+              <b-img src="./assets/images/madf.png" fluid alt="Make a Difference Foundation"></b-img>
+            </router-link>
+          </div>
+        </b-col>
+      </b-row>
+    </b-container>
+    <b-navbar toggleable="lg" variant="faded" type="light">
       <b-container>
-        <b-navbar-brand to="/">Site logo</b-navbar-brand>
-
-        <b-navbar-toggle target="nav-collapse"></b-navbar-toggle>
+        <b-navbar-toggle class="ml-auto" target="nav-collapse"></b-navbar-toggle>
 
         <b-collapse id="nav-collapse" is-nav>
           <b-navbar-nav>
             <b-nav-item to="/">Home</b-nav-item>
+            <b-nav-item to="">Medical missions</b-nav-item>
```

Comment:
I don't think we'll want the Medical Missions stuff in here, but we can talk about all that later when we are finalizing the design/layout.